stratisd_cert.py

```diff
@@ -823,13 +823,30 @@ def test_pool_set_fs_limit_too_low(self):
             "org.freedesktop.DBus.Error.Failed",
         )
 
+    @skip(_skip_condition(1))
+    def test_blockdev_set_userinfo_property(self):
+        """
+        Test setting the UserInfo property on a block device.
+        """
+        pool_name = p_n()
+        blockdev_path = StratisCertify.DISKS[0:1]
```

Comment:
Might as well get rid of ```blockdev_path``` identifier. Where you pass it as an argument to ```make_test_pool```, you might as well pass the value ```StratisCertify.DISKS[0:1]```

Reply:
Resolved.